LibTest/isolate/Isolate/pause_A01_t01.dart

```diff
@@ -22,34 +22,37 @@ import "dart:math";
 import "../../../Utils/expect.dart";
 
 // indefinitely running isolate
-entryPoint(SendPort sendPort){
+entryPoint(List<SendPort> sendPorts) {
+  sendPorts[0].send(42);
```

Comment:
After this send, the other side can theoretically receive the `42` message and pause this isolate **immediately**. 

Depending on the timing, the pause might happen before we even enter the loop below. The main isolate then would wait indefinitely to receive messages on `sendPorts[1]`.

The only effect of pausing an isolate is that it temporarily stops executing. Once resumed it will continue where it left off. So the only way to really measure if it was paused is to detect if there was a long pause (or use VM's `vm-service` protocol to ask for the isolate's state, but that's probably not a good idea for co19 tests).

The isolate itself can detect that with reasonably high probability, e.g. via something like this
```dart

const Duration pauseTime = const Duration(seconds: 10);

entryPoint(SendPort port) {
  final sw = Stopwatch()..start();
  bool notified = false;
  while (true) {
    if (!notified) {
      port.send('in-loop');
      notified = true;
    }
    if (sw.elapsed.inSeconds >= pauseTime.inSeconds) {
      // We were probably paused, otherwise this loop would not stop executing
      // for 10 seconds.
      port.send('detected-pause');
      break;
    } else {
      sw..reset()..start();
    }
  }
}

test() async {
  final onExit = ReceivePort();

  final rp = ReceivePort();
  final si = StreamIterator(rp);

  final isolate = await Isolate.spawn(entryPoint, rp.sendPort, onExit: onExit.sendPort);
  
  Expect.equals(true, await si.moveNext());
  Expect.equals('in-loop', si.current);

  final cap = isolate.pause();
  await Future.delayed(pauseTime);
  isolate.resume(cap);

  Expect.equals(true, await si.moveNext());
  Expect.equals('detected-pause', si.current);

  si.cancel();
  rp.close();

  await onExit.first;
}
```